sourcecred/sourcecred
# Bsodenkamp balanced policy lookback 2678

Creates a configurable lookback period for the balanced policy

Manual test plan:

* create an 'scprod' alias to call the 'main' sourcecred code
* set numIntervalsLookback to 0
* put debug outputs in the dev code to show all the values associated with balanced grain computations
* go to the cred instance
* run: scprod grain -s -f
* run: scdev grain -s -f
* make sure the standard output is the same and the debug outputs from 'scdev' make sense
* change the lookback to 100
* run: scdev grain -s -f
* make sure debug outputs make sense
* change lookback back to 0
* commit with debug outputs
* commit again without debug outputs
